src/synchronization/Synchronizer.js

```diff
@@ -247,41 +304,76 @@ function Synchronizer (event, handler) {
     that.updateDisableHandlers();
   };
 
-  // Removes an element as both a source and target
+  /**
+   * Remove an element from this synchronizer as both a target and source
+   *
+   * @param {HTMLElement} element - The element to be removed
+   */
   this.remove = function (element) {
     that.removeTarget(element);
     that.removeSource(element);
   };
 
-  // Returns the source elements
+  /**
+   * Get the array of source elements
+   *
+   * @returns {Array}
```

Comment:
For arrays please use typed definitions, e.g. here replace `{Array}` with `{HtmlElement[]}`.
This is so it renders correctly in our auto-generated api docs.

Similarly for other types, e.g. a string array is donated by `{string[]}`.

I'd be grateful if you could make this change throughout the PR :).

Reply:
I didn't even know that was a thing in JSDoc, so thanks for the heads up!